literally.cabal

```diff
@@ -1,11 +1,20 @@
 cabal-version: 3.0
 name: literally
 version: 0.2025.9.9
-synopsis: Convert type literals into values.
-description: Literally converts type literals into values.
+synopsis: Type-safe conversion of type literals into runtime values.
+description:
+  Literally is a minimal Haskell library that converts type literals into values
+  using type-safe mechanisms. It provides a single 'literal' function that
+  leverages Haskell's type system to convert compile-time literals (numbers,
+  strings, characters, booleans, tuples) into runtime values with compile-time
+  correctness guarantees. The library supports numeric types with automatic
+  bounds checking, string and character literals, boolean values, unit types,
+  and tuple types.
```

Comment:
This can only be viewed in a Haskell context, so it doesn't need to specify that it's a Haskell library.